enterprise/app/usage/usage.tsx

```diff
@@ -19,21 +25,64 @@ export default class UsageComponent extends React.Component<UsageProps, State> {
     this.setState({ loading: true });
     rpcService.service
       .getUsage(new usage.GetUsageRequest())
-      .then((response) => this.setState({ response }))
+      .then((response) => {
+        this.setState({ response, selectedPeriod: response.usage[0].period });
```

Comment:
any chance we could get a could not find `.period` of undefined

Reply:
That can happen if the configured usage start date is in the future. Updated the server code so that it just returns an empty month of usage for the current month in that case -- and also logs an error on server startup about the potential misconfiguration.

Reply:
might also be worth throwing a `?.period` in there to be safe in case the server ever changes

Reply:
Throwing an error in that case so that we only set the response if it's valid (and don't have to worry about edge cases anywhere in the rendering logic).